### ISSUE TITLE
Skipping split/going back twice

### ISSUE THREAD

**Issue by Vexcrate** [LiveSplit/LiveSplit#1859]
Whenever I use my hotkeys to skip a split/go back in splits, LiveSplit does this action twice.

**Comment by zeta12ti** [comment]
I can confirm that this still happens if the hotkeys contain modifiers. I use `` shift+` `` for my skip split hotkey and it consistently skips twice when using it. The same applies to undo, which I have bound to `` ctrl+` ``. Changing the hotkeys to not contain modifiers fixes the issue.

**Comment by just-ero** [comment]
Does this issue persist in the most recent stable version of LiveSplit?

**Comment by zeta12ti** [comment]
Yes, there do still seem to be problems in version 1.8.29 when skip split and undo split are set to be a modifier plus the same key as Start/Spit. I previously thought it was just the modifier, but it looks like it's actually having the same base key that causes issues.

Test 1:

Start/Split: `Oemtilde`, Undo Split: `Oemtilde. Shift`, Skip Split: `Oemtilde. Control`. 
Results: Start/Split works as expected. Undo Split undoes two splits. Skip Split skips two splits.

Test 2:

Start/Split: `Q. Shift`, Undo Split: `Q. Control`, Skip Split: `Q`. 
Results: Start/Split also skips a split. Undo Split undoes two splits, Skip Split works as expected.

Test 3:

Start/Split: `Q`, Undo Split: `W. Shift`, Skip Split: `T. Control`.
Results: All actions work as expected.
